File: 3014_minimum_number_of_pushes_to_type_word_i/Solution.cs
// LeetCode 3014 - Minimum Number of Pushes to Type Word I
// https://leetcode.com/problems/minimum-number-of-pushes-to-type-word-i/

public class Solution {
    public int MinimumPushes(string word) {
        int n = word.Length, ans = 0, k = 1;
        for (int i = 0; i < n / 8; i++) {
            ans += k * 8;
            k++;
        }
        ans += k * (n % 8);
        return ans;
    }
}

File: 3014_minimum_number_of_pushes_to_type_word_i/solution.py
# LeetCode 3014 - Minimum Number of Pushes to Type Word I
# https://leetcode.com/problems/minimum-number-of-pushes-to-type-word-i/


class Solution:
    def minimumPushes(self, word: str) -> int:
        n = len(word)
        ans = 0
        k = 1
        for i in range(n // 8):
            ans += k * 8
            k += 1
        ans += k * (n % 8)
        return ans
